autoload/ale/handlers/prettier.vim

```diff
@@ -0,0 +1,49 @@
+
+" Author: tunnckoCore (Charlike Mike Reagent) <mameto2011@gmail.com>
+" Description: Integration between Prettier and ESLint.
+
+" Here we use `prettier-eslint` intetionally,
+" because from v4 it is direct mirror of `prettier` - mimics
+" it's flags and etc.
+
+let g:ale_javascript_prettier_executable =
+\   get(g:, 'ale_javascript_prettier_executable', 'prettier-eslint')
+
+let g:ale_javascript_prettier_options =
+\   get(g:, 'ale_javascript_prettier_options', '')
+
+function! ale#handlers#prettier#GetExecutable(buffer) abort
+    if ale#Var(a:buffer, 'javascript_prettier_use_global')
```

Comment:
btw, locally it fails saying that "it is not set" or sort of like that.

@w0rp, it's the same when use the `eslint` fixer, so the problem isn't here.